### PR TITLE
refine GET_STARTED doc

### PR DESCRIPTION
**Pull request by jerrywgz**
refine GET_STARTED doc

see https://github.com/jerrywgz/models/blob/refine_GET_STARTED/PaddleCV/PaddleDetection/docs/GETTING_STARTED.md
see https://github.com/jerrywgz/models/blob/refine_GET_STARTED/PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md

### CODE REVIEW

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 1]
```diff
@@ -4,194 +4,136 @@ For setting up the running environment, please refer to [installation
...
-## Training
-
-#### Single-GPU Training
+## Training/Evaluation/Inference
 
+PaddleDetection provides scripots for training, evalution and inference with various features according to different configure.
 
 ```bash
-export CUDA_VISIBLE_DEVICES=0
+# specify python path
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: python -> Python

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md [hunk 2]
```diff
@@ -3,216 +3,155 @@
...
-
-
-#### 单GPU训练
+## 训练/评估/推断
 
+PaddleDetection提供了训练/训练/评估三个功能的使用脚本，支持通过不同可选参数实现特定功能
 
 ```bash
-export CUDA_VISIBLE_DEVICES=0
+# 指定python路径
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md]: 设置PYTHONPATH路径

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md [hunk 3]
```diff
@@ -3,216 +3,155 @@
...
-```bash
-export CPU_NUM=8
-export PYTHONPATH=$PYTHONPATH:.
-python tools/train.py -c configs/faster_rcnn_r50_1x.yml -o use_gpu=false
-```
+以下列表可以通过`--help`查看
+
+|         FLAG             |     支持脚本    |        用途        |      默认值       |         备注         |
+| :----------------------: | :------------: | :---------------: | :--------------: | :-----------------: |
+|          -c              |      ALL       |  配置文件路径  |  None  |  **完整配置说明请参考[配置案例](config_example)** |
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md]: 配置文件路径 ->   指定配置文件

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md [hunk 4]
```diff
@@ -3,216 +3,155 @@
...
-export PYTHONPATH=$PYTHONPATH:.
-python -u tools/train.py -c configs/faster_rcnn_r50_1x.yml --eval
+python -u tools/train.py -c configs/faster_rcnn_r50_1x.yml --eval -d dataset/coco
 ```
 
-可通过设置`--eval`在训练epoch中交替执行评估, 评估在每个snapshot\_iter时开始。可在配置文件的`snapshot_iter`处修改。
-如果验证集很大，测试将会比较耗时，影响训练速度，建议减少评估次数，或训练完再进行评估。
-当边训练边测试时，在每次snapshot\_iter会评测出最佳mAP模型保存到
-`best_model`文件夹下，`best_model`的路径和`model_final`的路径相同。
+在训练中交替执行评估, 评估在每个snapshot\_iter时开始。每次评估后还会评出最佳mAP模型保存到`best_model`文件夹下。
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md]: 看展示效果 https://github.com/jerrywgz/models/blob/refine_GET_STARTED/PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md#%E6%A8%A1%E5%9E%8B%E8%AE%AD%E7%BB%83

觉得调整下58行的缩进吧

下同，每个小结都调整下缩进。

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md (hunk 1 above)
Comment (jerrywgz) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: Done

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md (hunk 2 above)
Comment (jerrywgz) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md]: Done

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md (hunk 3 above)
Comment (jerrywgz) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md]: Done

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md (hunk 4 above)
Comment (jerrywgz) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED_cn.md]: Done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**